swagger2locustio/__main__.py

```diff
@@ -14,14 +14,27 @@ def main():
     """Launching function"""
 
     parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-    parser.add_argument("--swagger-file", help="path to swagger file", required=True, type=Path)
+    parser.add_argument("-f", "--swagger-file", help="path to swagger file", required=True, type=Path)
     parser.add_argument(
-        "--results-path", help="path to store locustfile.py", required=False, default=Path("generated"), type=Path,
+        "-r",
+        "--results-path",
+        help="path to store locustfile.py",
+        required=False,
+        default=Path("generated"),
+        type=Path,
     )
     parser.add_argument(
-        "--loglevel", help="logging level", required=False, default="info", choices=settings.LOGGING_LEVELS, type=str,
+        "-v",
+        "--verbose",
+        nargs="*",
+        help="logging level",
+        required=False,
+        default="info",
+        choices=settings.LOGGING_LEVELS,
+        type=str,
     )
     parser.add_argument(
+        "-s",
         "--strict-level",
```

Comment:
There is no use of --strict-level in our code. We need to think about how it can be used.

Reply:
`swagger_strategy = BaseStrategy(swagger_data, args.results_path, mask, args.strict_level)`